Framework/Kernel/src/ProgressText.cpp

```diff
@@ -5,14 +5,17 @@
 //     & Institut Laue - Langevin
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MantidKernel/ProgressText.h"
-
+#include "MantidKernel/Logger.h"
```

Comment:
Could I ask that we actually remove this file/class and associated files completely. It's not actually used any longer.

Reply:
removing this seems to cause ThreadPoolTest to fail building

Reply:
nevermind, I think I've figured it out